src/App.js

```diff
@@ -1,29 +1,7 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] kindly ensure that your components are class based. 
example
```
class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

Reply:
Done ✅